src/components/tabs.tsx

```diff
@@ -6,41 +6,97 @@ import {
   TabPanel as UnstyledTabPanel
 } from "react-tabs";
 
+enum SectionTypes {
+  BLOCKS = "blocks",
+  CODE = "code",
+  CONTROLS = "controls"
+}
+type TabInfo = {
+  [tab in SectionTypes]: {
+    name: string;
+    index: number;
+    backgroundColor: string;
+    hoverBackgroundColor: string;
+  }
+};
+const kTabInfo: TabInfo = {
```

Comment:
I'm open to naming convention ideas here if people don't like `kTabInfo` (this is convention in CLUE and some of our other apps).